frontend/src/pages/DownloadingScreen.vue

```diff
@@ -2,9 +2,9 @@
   <div class="container">
     <div class="row">
       <div class="col-md-6 mx-auto text-center">
-        <p v-if="downloading.filename !== ''" >
+        <!-- <p v-if="downloading.filename !== ''" >
```

Comment:
I didn't mean to delete this HTML section but the actual file it downloads so that you can test it downloading again. We need to improve the visual alignment of the text on this page. maybe make it all "centered"?